src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareContracts/Component/TransportMode.tsx

```diff
@@ -16,10 +38,19 @@ const TransportMode = ({
   disabled?: boolean;
 }) => {
   const styles = useStyles();
+  const {theme} = useTheme();
+  const fontScale = useFontScale();
   const {t} = useTranslation();
+
+  const modesCount: number = modes.length;
+  const modesToDisplay = modes.slice(0, modesDisplayLimit);
+  const boxHeight = {
+    height: theme.icon.size['small'] * fontScale + theme.spacings.xSmall * 2,
+  };
```

Comment:
Hmm. I feel like maybe this can be avoided? Why not just let the icon scale/text scale. The padding doesn't need to scale.

Reply:
When I was testing this, I removed the `boxHeight` is not applied, the text grows and the box looks much higher than the transport icons. So seems like doing these constraints, the text inside to match the height.